Code/Tools/ProjectManager/Source/ProjectExportWorker.cpp

```diff
@@ -213,7 +225,31 @@ namespace O3DE::ProjectManager
             return AZ::Failure(error);
         }
 
-        return AZ::Success();
+        // Fetch the output directory if the process was successful
+        // All standardized O3DE scripts end with a log line stating the output directory in quotes as the last word
+        // Use regex to collect that directory, and collect the slice of the word surrounded in quotes
+        QString exportOutputLastLine = exportOutput.split('\n', Qt::SkipEmptyParts).takeLast();
+
+        QRegExp quotedDirRegex("'.*'");
+        int pos = quotedDirRegex.indexIn(exportOutputLastLine);
+        if (pos > -1)
+        {
+            QStringList entries = quotedDirRegex.capturedTexts();
+            QString outputDirQuoted = entries.takeLast();
+            QString outputDir = outputDirQuoted.mid(1, outputDirQuoted.size()-2);
+            if (QDir(outputDir).isAbsolute())
+            {
+                m_expectedOutputDir = outputDir;
+                return AZ::Success();
+            }
+            else
+            {
+                m_expectedOutputDir = "";
+            }
+        }
```

Comment:
1. This is an inefficient way to parse out the output path. It doesn't take full advantage of regular expression captures, because its still doing additional string processing after the capture (this adds additional string allocation unnecessarily).

```
        QRegExp projectExportPathRegex("Project Exported to '(.*)'.");
        int pos = projectExportPathRegex.indexIn("[INFO]  Project Exported to 'f:\\foo\\bar\\'. !!");
        if (pos > -1)
        {
            m_expectedOutputDir = reg.cap(1);
        }
```
2. Being able to capture the output folder does not necessarily indicate a failure of the whole process. It should not return an error if it cannot find/parse the output folder.
3. Assuming that the very last line in the log could be problematic if someone updates the project export script to log out additional information after the fact, (since there is nothing in the code that says "this must be logged last")

Reply:
point #3 was the main reason I originally preferred the pythonbindings c++ approach, as that was the more direct way of correctly fetching the intended output path for each platform.

That being said, I dont think there is any good way to mechanically enforce that, other than to add a documentation comment stating "this must be logged last"

Reply:
but if the regex is updated to a fully unique phrase, that concern will likely go away

Reply:
If you follow a stricter pattern, and use regex properly, you don't have to just scan the last line. (see the comment about splitting the lines).  If you take the C++ python bindings approach, then that seems like more work than needed especially for a bug fix.

Reply:
It may be worth it to do the regex search inside the loop so it scans every log line.

Reply:
actually if the regex is specific enoguh I dont think it's necessary to search inside the loop. If all of the standard output is read, we only need to ensure the project export log line is unique enough.

Reply:
Well, are you guaranteed that the last chunk of `exportOutput` will contain the export log line? My guess that will always be the case, but its still a possibility.

Reply:
It's a reasonable constraint to make for the export worker considering we only provide options for the standard o3de scripts. 

that means we can force the definition to make sure the export log line is present (but I prefer just making the regex specifically look for a unique project export log line which should be present in all standard export scripts).